README.md

```diff
@@ -176,6 +176,31 @@ It also is possible to check for realm roles and application roles.
 * `hasRole('realm:admin')` will check the logged in user has the admin realm role
 * `hasRole('some-other-app:admin')` will check the loged in user has the admin realm role in a different application
 
+### Error Codes
+
+Library will return specific GraphQL errors to the client that can
+be differenciated by using error codes.
+
+Example response from GraphQL Server could look as follows:
+
+```json
+{
+   "errors":[
+      {
+        "message":"User is not authorized. Must have one of the following roles: [admin]",
+        "code": "FORBIDDEN"
+      }
+   ]
+}
+```
+
+Possible error codes: 
+
+- `NOT_AUTHENTICATED`: returned when user is not authorized to access API because it requires login
```

Comment:
```suggestion
- `NOT_AUTHENTICATED`: returned when user is not authenticated to access API because it requires login
```

Reply:
Would it make sense to maybe change this to `UNAUTHENTICATED`. This is the same code Apollo Server uses. We might as well try keep things similar to how it's done in the ecosystem https://www.apollographql.com/docs/apollo-server/data/errors/#codes